### ISSUE TITLE
Project description doesn't adequately explain the 'raison d'être' of this project

### ISSUE THREAD

**Issue by dchambers** [jednano/ts-compiler#1]
The tag line for this project is "FAST compilation of TypeScript source or file(s)", yet there is no indication of what makes it faster than the underlying TypeScript APIs; for example, does it enable caching support so that only changed files are re-compiled?

If this project merely offers a simpler API than `new ts.BatchCompiler.compile(...)`, then it would be really helpful to clearly state that in the introductory text.

**Comment by jednano** [comment]
My memory isn't the best, so I don't remember all the details, but I had issues compiling TypeScript files with other tools I had been using, in that it would take roughly 2 seconds per file, regardless of complexity. If I recall, it's because those tools had a compile method, but didn't expose a BatchCompiler. My tool's compile method uses the BatchCompiler's compile method, regardless of whether you are compiling one file or n number of files. This is what makes it faster.

The project tagline has been updated to reflect this.

Furthermore, in case you didn't know, from the TypeScript team, "exposing a cleaner, more usable API is on our backlog for post v1" [(source)](https://typescript.codeplex.com/workitem/2078).

As it stands now, exposing the TypeScript API is a hack. It's not yet intended for public consumption and I'm eagerly awaiting the insignificance of this tool I have created.
